server/test/swagger_2/swagger_2.test.js

```diff
@@ -0,0 +1,99 @@
+const supertest = require('supertest');
+
+const app = require('../../app');
+const request = supertest(app);
+
+const jsonResponse1 = require('./data/output/jsonResponse1.json');
+const jsonResponse2 = require('./data/output/jsonResponse2.json');
+const { yamlResponse1, yamlResponse2 } = require('./data/output/yamlResponses');
+
+
+describe('POST /api/specification/swagger_2/openapi_3', () => {
+  //Swagger 2 (JSON) to OpenAPI 3 (JSON)
+  it('should convert swagger_2 (json) to openapi_3 (json) ', async () => {
+    const res = await request
+      .post('/api/specification/swagger_2/openapi_3')
+      .attach('file', 'test/swagger_2/data/input/swagger_2.json');
+    expect(res.status).toBe(200);
+    expect(JSON.parse(res.text)).toEqual(jsonResponse1);
+    expect(res.body).toEqual({});
+  });
+
+  //Swagger 2 (JSON) to OpenAPI 3 (YAML)
+  it('should convert swagger_2 (json) to openapi_3 (yaml) ', async () => {
+    const res = await request
+      .post('/api/specification/swagger_2/openapi_3')
+      .attach('file', 'test/swagger_2/data/input/swagger_2.json')
+      .query({'targetSyntax': 'yaml'});
+    expect(res.status).toBe(200);
+    expect(res.text).toEqual(yamlResponse1);
+    expect(res.body).toEqual({});
+  });
+
```

Comment:
Where are the negative cases being tested? @pcgamer1 
400 for invalid extension, 404 for mapping not found, 400 for couldn't parse file and convert?